lib/wowrb.rb

```diff
@@ -161,10 +161,61 @@ def self.credentials
   end
   private_class_method :credentials
 
+  def self.check
+    if ENV['BATTLE_NET_KEY'].nil?
```

Comment:
what happen if `ENV['BATTLE_NET_KEY']` is empty and `ENV['BATTLE_NET_REGION']` is empty too? And if locale empty too? Which message will we get?